editor/scene_tree_dock.cpp

```diff
@@ -4712,7 +4710,7 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	filter = memnew(LineEdit);
 	filter->set_h_size_flags(SIZE_EXPAND_FILL);
 	filter->set_placeholder(TTRC("Filter: name, t:type, g:group"));
-	filter->set_accessibility_name(TTRC("Filter"));
+	filter->set_accessibility_name(TTRC("Filter: name, t:type, g:group"));
```

Comment:
Oh... This may turn into a jumble of words when read by a screen reader? Not that I was ever a fan of the way https://github.com/godotengine/godot/pull/81675 changed the placeholder, but maybe it's best to say "Filter" for accessibility, for now.

Reply:
Changed it to "Filter Nodes".